R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,105 @@
+# importCNA.GISTIC2.R
+#
+#
+#'@function Imports copy number aberration data from Firehose database,
+# produces an rCNA object saved as a RDS file.
+#
+#
+#
+#'@param fFHG Local fully qualified path of Firehose GISTIC2 CNA file.
+#'@param dCNA The local fully qualified path of a directory to store the rCNA RDS file(s).
+#'@param silent Boolean. Default: FALSE. Whether or not to write progress information to console.
+#'@param noLog Boolean. Default: FALSE. Whether or not to log results.
+#'@return gG object
+#'@notRun {
+#'@family importM.COSMIC, importM.TCGA, importM.GISTIC2, importM.ProjectGenie
+#}
+#'@example
+# fFHG <- "inst/extdata/devCNA.txt"
+# dCNA <- "inst/extdata/dCNA"
+# silent <- FALSE
+# writeLog <- TRUE
+# importCNA.GISTIC2("inst/extdata/devCNA.txt", "inst/extdata/dCNA")
+#'@export
+#
+#
+#
+
+importCNA.GISTIC2<- function(fFHG, dCNA, silent = FALSE, writeLog = TRUE){
+
+    # check if file exists
+
+    if(!file.exists(fFHG)){
+            stop(fFHG)
+    }
+
+    # parameter check
+
+    cR <- character()
+    cR <- c(cR, .checkArgs(fFHG,        like = "devCNA.txt",   checkSize = TRUE))
```

Comment:
I don't think line 32-34 is necessary if you also do a checkargs checking if it's a file